test/constant.ts

```diff
@@ -18,3 +18,9 @@ export const KIBANA_SERVER_PASSWORD: string = 'kibanaserver';
 
 export const ELASTICSEARCH_VERSION: string = '7.8.0';
 export const SECURITY_ES_PLUGIN_VERSION: string = '1.9.0.0';
+
+export const ADMIN_USER: string = 'admin';
+export const ADMIN_PASSWORD: string = 'admin';
```

Comment:
Minor:  probably use a different value just in case the wrong const variable is misplaced.